.github/workflows/cibuildwheel.yml

```diff
@@ -0,0 +1,113 @@
+name: Build CI wheels
+
+on:
+  push:
+    branches:
+      - master
+      - v[0-9]+.[0-9]+.x
+      - cibuildwheel
+    tags:
+      - v*
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==1.6.3
```

Comment:
This is amazing!! Why are we pinned at 1.6.3 though?

Reply:
That's because matplotlib pinned it 😂 

https://github.com/matplotlib/matplotlib/blob/2303c2b4afa7ecd1c3c576377c77dc53e0c86273/.github/workflows/cibuildwheel.yml

I'd like to take an "if it ain't broke" approach here for now. We can become adventurous after 0.18?

Reply:
ok

Reply:
`1.6.4` is the most recent version and `1.6.3` dates back to Oct 12, so quite recent.